source/index-glossary.rst

```diff
@@ -1,11 +1,6 @@
-.. _pmm/glossary/all:
+:orphan:
```

Comment:
Old glossary index page that introduced an unnecessary level. Now redirected to new glossary.